pytorch_lightning/accelerators/accelerator.py

```diff
@@ -227,29 +241,30 @@ def get_reference_model(self, model) -> LightningModule:
 
     def __getstate__(self):
         return {
-            'trainer': self.trainer,
-            'nickname': self.nickname,
-            'cluster_environment': self.cluster_environment,
-            'dist': self.dist,
-            'ddp_plugin': self.ddp_plugin
+            "trainer": self.trainer,
+            "nickname": self.nickname,
+            "cluster_environment": self.cluster_environment,
+            "dist": self.dist,
+            "ddp_plugin": self.ddp_plugin,
```

Comment:
can you add `-S` to avoid string normalization here, in case you are using black?? These are unrelated changes to this PR.

Reply:
@rohitgr7 yes